.travis.yml

```diff
@@ -1,13 +1,20 @@
 language: node_js
 node_js:
   - "node"
+  - "10"
   - "8"
   - "6"
 
-# Cache dependencies in $HOME/.yarn-cache across builds
-cache: yarn
```

Comment:
Is this no longer needed?

Reply:
The issue I fixed with #32 had to do with the node_modules installed for Node 6/8 not working with Node 10. So we can no longer share dependencies across builds as a result because different dependencies get installed depending on Node version. I think it's safer this way...